java/src/main/native/ai_onnxruntime_OnnxTensor.c

```diff
@@ -71,6 +71,7 @@ JNIEXPORT jlong JNICALL Java_ai_onnxruntime_OnnxTensor_createTensorFromBuffer
 
     // Extract the buffer
     void* bufferArr = (*jniEnv)->GetDirectBufferAddress(jniEnv,buffer);
+    bufferArr = bufferArr + bufferPos;
```

Comment:
>  bufferArr = bufferArr + bufferPos [](start = 2, length = 35)

CI is having an issue here.void* can not be advanced since the size of the data type is not known.
C:\a\1\s\java\src\main\native\ai_onnxruntime_OnnxTensor.c(74,38): error C2036: 'void *': unknown size [C:\a\1\b\RelWithDebInfo\onnxruntime4j_jni.vcxproj]

Reply:
Hmm, ok, must be a Visual C++ thing, it compiled fine with clang on my Mac. I'll fix it Monday.

Reply:
You also may want to merge the master since there are WInML warning fixes that may be affecting this CIs as well. I don't know how they made it to the master.

---
In reply to: [491230183](https://github.com/microsoft/onnxruntime/pull/5223#discussion_r491230183) [](ancestors = 491230183)

Reply:
I pulled from master and rebased my branch and now I get a compile error out of graph.cc (which I've not touched).
```/Users/apocock/Development/onnxruntime/onnxruntime/core/graph/graph.cc:584:22: error: loop variable 'node_arg_name' is always a copy because the range of type 'const
      flatbuffers::Vector<flatbuffers::Offset<flatbuffers::String> >' does not return a reference [-Werror,-Wrange-loop-analysis]
    for (const auto& node_arg_name : *fbs_node_arg_names) {
                     ^
/Users/apocock/Development/onnxruntime/onnxruntime/core/graph/graph.cc:584:10: note: use non-reference type 'const flatbuffers::String *'
    for (const auto& node_arg_name : *fbs_node_arg_names) {
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
I bodged around that on my machine (and didn't commit that bit), seems like something in XCode 12 isn't happy with that bit of code. Anyway. I've added a cast so the pointer arithmetic is hopefully functional on Windows, could you rerun the CI?